### DISCUSSION TITLE
Interactive Graph in the Brave Browser

### DISCUSSION THREAD

**Discussion by mettaseva** [oleeskild/obsidian-digital-garden#101] [Q&A] [answered]
@oleeskild ~ 

After working with my new site in my Brave browser, I discovered the performance of the interactive graph is not to up to par with its performance in the Edge browser:
- navigation, when it works, is slower and
- often the navigation doesn't work at all.

Is this an issue that can be resolved without too much difficulty?

**Comment by oleeskild** [comment, marked as answer]
This has been answered in #103 :)

**Reply by mettaseva** [reply]
Thanks! Very much appreciate your prompt and helpful feedback. :-)
